apps/glusterfs/app_block_volume_test.go

```diff
@@ -203,7 +203,7 @@ func TestBlockVolumeLargerThanBlockHostingVolume(t *testing.T) {
 	// blockhosting volume and any new blockhosting volume that can
 	// be created
 	request = []byte(`{
-        "size" : 1600
+        "size" : 1079
```

Comment:
Why this change in size?

Reply:
I want to test the corner case, to utilize the completely available free size

Reply:
ah, ok, that's the block volume request size not the BHV - I read this wrong :-)
Makes sense.